server/job.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"github.com/google/uuid"
+	pb "github.com/romanlevin/felek/jobs"
+	"os/exec"
+)
+
+type job struct {
+	id        string
+	cmd       *exec.Cmd
+	stopped   bool
+	exitError string
+	owner     string
+}
+
+func jobStatus(j *job) *pb.JobStatus {
+	c := j.cmd
+	if j.exitError == "" && c.ProcessState != nil {
```

Comment:
I'm not entirely sure if it can race here, but I'm moving to a per-job lock as per @bernardjkim's suggestion, and there we will acquire a read lock in `jobStatus`, and a write lock when stopping a job, which should prevent races here.